model/src/main/kotlin/utils/FindingsMatcher.kt

```diff
@@ -67,22 +79,47 @@ class FindingsMatcher(
             .distinct()
 
     /**
-     * Return those statements in [copyrights] which are in the vicinity of [licenseStartLine] as defined by
-     * [toleranceLines].
+     * Return the line range in which copyright statements should be matched against the license finding at the
+     * location given by [licenseStartLine] and [licenseEndLine].
```

Comment:
Please also explain the `copyrightLines` parameter.

Reply:
Done.